tests/graph/test_op.py

```diff
@@ -232,3 +232,55 @@ def perform(self, *_):
 
     x = pt.TensorType(dtype="float64", shape=(1,))("x")
     assert SomeOp()(x).type == pt.dvector
+
+
+def test_op_name():
+    op_name = "op_name"
+
+    class DummyType(Type):
+        def filter(self, data):
+            return data
+
+        def __eq__(self, other):
+            return isinstance(other, DummyType)
+
+        def __hash__(self):
+            return hash(DummyType)
+
+        def __repr__(self):
+            return "DummyType()"
```

Comment:
Do we need a new dummy type? Can we reuse one from the existing ones for testing?

Reply:
The  [existing dummy type](https://github.com/pymc-devs/pytensor/blob/main/tests/graph/test_op.py#L23-L25) takes an argument called: `thingy`

I'm not sure if that should be reused (I assume it has a specific use)

Reply:
Should be fine, it's a test Type, it takes an argument for other purposes but that shouldn't be problematic for us. Just pass None or whatever you want